reviewer_readme.md

```diff
@@ -1,46 +1,116 @@
+You've been chosen to review article submissions from Github for Intel's Android Hub. This guide will help you through the things you need to do, in order to accept the article with confidence.
+
+--------
+
+## Process Overview
+
+![process](https://drive.google.com/uc?id=0BwL_X3dS1kN_QVNwYXRydVR6VGs)
+
+--------
+
+### Step 1: Review Open Pull Requests
+
+Open the list of article submissions: https://github.com/intel-android/androidhub/pulls
+
+![pull requests](https://drive.google.com/uc?id=0BwL_X3dS1kN_Q3hRRFVGS2VMWWc)
+
+All submissions history is preserved and openly viewable. Clicking on a pull request (PR) will open that PR's details. Also appearing next to the title of the pull request is either a green checkmark or a red X indicating whether tests have passed.
+
+The PR details show you the files changed by the author, the author's commentary, and the pass/fail result of the build test. Review the files changed and follow these simple guidelines:
+
+##### Acceptable Changes:
+
+- `~/data/authors/<author>.yaml`
+- anything in their posts folder inside `~/posts/<author>`
+
+Changes to these folders should be considered for approval.
+
+##### Unacceptable Changes:
+
+- `assets` - front end css, js, and images
+- `scripts` - support scripts and helpers for the site
+- `spec` - testing suite for the site
+- `views` - templates for the site
+
+Changes to these folders should be rejected immediately.
+
+###### Tip: [Learn more about pull requests](https://help.github.com/articles/using-pull-requests/#reviewing-the-pull-request) from the masters.
 
-&nbsp;
+--------
 
-## Goal
-- View open pull requests/submissions
-- Preview a submission
-- Run tests
-- Accept/Deny
+### Step 2: Preview submission
 
-#### View Open Pull Requests
-![preview](https://drive.google.com/uc?id=0B6rG4e8CXFiGWmcwMHNhdUFOdk0)
-Direct Link: https://github.com/intel-android/androidhub/pulls
+###### Setup
 
-This page will have a list of article submissions. I suggest playing around here and learning what's going on. For example, you can view old submissions, rejected submissions, etc. All the history is preserved and openly viewable for anyone. By clicking on a pull request (PR), you'll find yourself at the PR detail.
+Don't know where to start? Reference our helpful [Setup Guide](readme.md).
 
-The PR detail will show you all the files changed by the author, with commentary, as well as whether or not the submission is passing the build test. Review the changed files within the Pull Request. Only the author's own folder should be changed. Nothing in ~/assets, ~/scripts/, ~/spec, ~/views should change either. Reject the Pull Request right away if the author attempted to change anything outside their own folder in ~/posts or their author.yaml file in ~/data/authors.
+###### Preview the submission
 
-### Automated Testing
+1. Click "open this in GitHub Desktop" on the pull request page.
```

Comment:
Since #1 in this section is “open this in GitHub Desktop”, can we clearly state in “setup” that they’ll need the desktop client installed to proceed? I know we reference the setup guide, but likely folks will skip that. It's worth a simple disclaimer that they'll need the desktop client. 

Lets include links for PC/Mac download and ensure all instructions throughout this doc cover both platforms. Both of our primary clients are on Windows.